www/src/components/pdf/PDFViewer.tsx

```diff
@@ -0,0 +1,31 @@
+import { Grid } from '@material-ui/core';
+import React, { FC } from 'react';
+import { useEffect } from 'react';
+
+import ViewSDKClient from './ViewerSDKClient';
+
+// TODO: Modify to properly pass necessary props to make get resume from api
```

Comment:
I'm not really sure what this TODO is saying?

Reply:
Oh yeah so it was just a note to pass in any props necessary to integrate with the api. For instance a user id but I wasn't too sure yet how this component will access the pdf from the backend. I want to do that all in this PR tbh so I think I'll ping you to discuss how the pdf stuff will work.